src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -724,6 +726,7 @@ private List<LaunchCommand> GetInitializeCommands()
                     }
 
                     commands.Add(new LaunchCommand("-break-insert main", ignoreFailures: true));
+                    this._deleteEntryPointBreakpoint = true;
```

Comment:
I wonder if we should instead install a command handler here so we can find the bpno if the '-bread-insert' succeeds. There is a slight risk of deleting some other random thing that we don't understand otherwise.